src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -276,10 +276,12 @@ def test_ad_field_init_wrong_type():
 
 
 def test_ad_field_init_wrong_iteration_space():
-    ''' Test that an error is raised if a wrong iteration
-    space (other than ['cells', 'dofs']) is passed to the
+    ''' Test that an error is raised if a wrong iteration space
+    (other than ['cell_column', 'dof']) is passed to the
     LFRicArgDescriptor._init_field() method.
 
+    TODO #870 update this test.
```

Comment:
Please add a bit more information on how specifically this test needs to be updated.

Reply:
Done (although, on reflection, this comment isn't really necessary as this test will just fail when #870 is done).